portal/static/js/profile.js

```diff
@@ -1633,6 +1633,25 @@
                     });
                 });
             },
+            getNoOrgDisplay: function() {
+                return "<p class='text-muted'>"+this.modules.i18next.t("No affiliated clinic")+"</p>";
+            },
+            getOrgsDisplay: function() {
+                if (!this.demo.data.careProvider || this.demo.data.careProvider.length === 0) {
+                    return this.getNoOrgDisplay();
+                }
+                /* example return from api demographics: [{ display: Duke, reference: "api/organization/1301"}, {"display":"Arvin George","reference":"api/practitioner/1851648521?system=http://hl7.org/fhir/sid/us-npi"}]
+                 * NOTE: need to exclude displays other than organization */
+                var self = this;
+                var arrDisplay = this.demo.data.careProvider.map(function(item) {
+                    if (item.reference.match(/^api\/organization\/0$/gi)) { //organization id = 0
```

Comment:
If you're doing an exact match for that string (`api/organization/0`) do you still need a regular expression?